### PR TITLE
Alert Dialog

### PR DESCRIPTION
**Pull request by Abhishek-Acme**

### CODE REVIEW

File: playground/lib/widgets/playground_alert_dialogs.dart [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+            showDialog(
+              context: context,
+              builder: (context) => const FirstAlertDialog(
+                title: Text('Lorem ipsum dolor sit amet.'),
+                content: Text(
+                    'Lorem ipsum dolor sit amet. 33 aperiam rerum vel sint veniam nam dolor placeat! Aut quis sint aut repellat dolorum 33 minus accusamus.'),
+              ),
+            );
+          },
+          child: const Text('Show Dialog')),
```
Comment (sarbagya-acme) [inline playground/lib/widgets/playground_alert_dialogs.dart:21]: missing trailing comma

File: lib/src/core/widgets/alert_dialog.dart [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+        clipBehavior: config.clipBehavior,
+        actionsOverflowButtonSpacing: config.actionsOverflowButtonSpacing,
+        actionsOverflowDirection: config.actionsOverflowDirection,
+        actionsPadding: config.actionsPadding,
+        buttonPadding: config.buttonPadding,
+        contentPadding: config.contentPadding,
+        contentTextStyle: config.contentTextStyle,
+        iconColor: config.iconColor,
+        iconPadding: config.iconPadding,
+        insetPadding: config.insetPadding as EdgeInsets,
```
Comment (sarbagya-acme) [inline lib/src/core/widgets/alert_dialog.dart:48]: why was this needed ?

**Review by sarbagya-acme — CHANGES_REQUESTED** [review]

File: lib/src/core/widgets/alert_dialog.dart (hunk 2 above)
Comment (Abhishek-Acme) [reply lib/src/core/widgets/alert_dialog.dart:48]: we can only encode or decode to EdgeInsetsGeometry and not EdgeInsets

**Review by sarbagya-acme — APPROVED** [review]
